### PR TITLE
an official beta2 for official theano 0.8.0

### PR DESCRIPTION
**Pull request by stonebig**

### CODE REVIEW

**Comment by stonebig** [comment]
making it possibly happen: https://github.com/pymc-devs/pymc3/issues/1027

**Comment by springcoil** [comment]
Does this not need a dependency added - I guess that's not really necessary, since >= 0.7 is probably enough for Theano. Thanks for showing ownership here @stonebig. I think this can be merged - any objections Thomas?

File: setup.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 AUTHOR_EMAIL = 'chris.fonnesbeck@vanderbilt.edu'
 URL = "http://github.com/pymc-devs/pymc"
 LICENSE = "Apache License, Version 2.0"
-VERSION = "3.0"
+VERSION = "3.0.0b2"
```
Comment (twiecki) [inline setup.py]: Lets stick with `"3.0b2"`.

**Comment by twiecki** [comment]
We would also need a list of added features / bug-fixes. E.g. https://github.com/quantopian/pyfolio/blob/master/WHATSNEW.md

**Comment by springcoil** [comment]
We really should have this happen while people add Pull Requests. 

What do you think the highlights are.

**Comment by twiecki** [comment]
Agreed. Well, there was a lot of work happening. Best to go through the commit / merge history and look for the big PRs. PPC is one, a big NUTS bug-fix, many speed-ups, auto-transforms.

**Comment by springcoil** [comment]
Do I just look at the merge history manually or is there a neat way to do this?

**Comment by twiecki** [comment]
Not really, https://github.com/pymc-devs/pymc3/issues?q=is%3Aissue+is%3Aclosed might have a better presentation of the highlights.

**Comment by twiecki** [comment]
Sorry, I meant https://github.com/pymc-devs/pymc3/pulls?q=is%3Apr+is%3Aclosed.

**Comment by springcoil** [comment]
Might get some time tomorrow - but don't let me stop someone else doing it.

File: setup.py [hunk 2]
```diff
@@ -12,7 +12,7 @@
 AUTHOR_EMAIL = 'chris.fonnesbeck@vanderbilt.edu'
 URL = "http://github.com/pymc-devs/pymc"
 LICENSE = "Apache License, Version 2.0"
-VERSION = "3.0.0b2"
+VERSION = "3.0b2"
```
Comment (springcoil) [inline setup.py:15]: LGTM :+1:

**Comment by twiecki** [comment]
Theano `0.8.1` is released now, we should probably peg to that.

**Comment by stonebig** [comment]
You have something that doesn't work on Theano 8.0.0 ? Maybe there is an hidden bug in 0.8.1.

**Comment by twiecki** [comment]
Yes, there are quite a few fixes we rely on.

**Comment by twiecki** [comment]
@springcoil Any news on the release notes?

**Comment by stonebig** [comment]
will we have that beta2 before pycon 2016 ?

**Comment by springcoil** [comment]
Yeah that got sidelined because of life. Will try again soon

**Comment by twiecki** [comment]
@springcoil any update on release notes?

**Comment by springcoil** [comment]
Thomas,
I pushed some stuff to changelog - do I need to do something else?

On Wed, May 25, 2016 at 5:27 PM, Thomas Wiecki notifications@github.com
wrote:

> @springcoil https://github.com/springcoil any update on release notes?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> https://github.com/pymc-devs/pymc3/pull/1043#issuecomment-221626856

## 

Peadar Coyle
Skype: springcoilarch
www.twitter.com/springcoil
peadarcoyle.wordpress.com

**Comment by springcoil** [comment]
Hi Thomas,
Do you want something like - by commit etc and what it involves.

On Wed, May 25, 2016 at 5:28 PM, Peadar Coyle peadarcoyle@googlemail.com
wrote:

> Thomas,
> I pushed some stuff to changelog - do I need to do something else?
> 
> On Wed, May 25, 2016 at 5:27 PM, Thomas Wiecki notifications@github.com
> wrote:
> 
> > @springcoil https://github.com/springcoil any update on release notes?
> > 
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly or view it on GitHub
> > https://github.com/pymc-devs/pymc3/pull/1043#issuecomment-221626856
> 
> ## 
> 
> Peadar Coyle
> Skype: springcoilarch
> www.twitter.com/springcoil
> peadarcoyle.wordpress.com

## 

Peadar Coyle
Skype: springcoilarch
www.twitter.com/springcoil
peadarcoyle.wordpress.com

**Comment by twiecki** [comment]
Changelog looks great. I think we can move ahead with the beta2.

**Comment by springcoil** [comment]
Cool

**Comment by twiecki** [comment]
Closing as we have a RC1 now.

**Pull request closed without merging**